src/components/PostDetails.tsx

```diff
@@ -132,7 +92,7 @@ export const PostDetails: React.FC<Props> = ({ post }) => {
                     type="button"
                     className="delete is-small"
                     aria-label="delete"
-                    onClick={() => deleteComment(comment.id)}
+                    onClick={() => dispatch(removeComment(comment.id))}
                   >
                     delete button
```

Comment:
I know it is initial styles, but would be nice to fix them anyway) Please, do it next time 🙂